db/migrate/20221109105200_change_sample_concentrations_to_single_value.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeSampleConcentrationsToSingleValue < ActiveRecord::Migration
+
+  def up
+    Sample.find_each do |sample|
+      if sample.core_fields.key?("concentration_number") && sample.core_fields.key?("concentration_exponent")
+        sample.core_fields["concentration"] = sample.core_fields["concentration_number"].to_i * (10**(sample.core_fields["concentration_exponent"].to_i))
+        sample.core_fields.delete("concentration_number")
+        sample.core_fields.delete("concentration_exponent")
+        sample.save
+      else
+        sample.core_fields["concentration"] = 0
+        sample.save
```

Comment:
I don't think we want to set a default concentration for all samples. Let's leave them as `nil`.